scipy/stats/_resampling.py

```diff
@@ -315,7 +329,17 @@ def bootstrap(data, statistic, *, n_resamples=9999, batch=None,
     Parameters
     ----------
     data : sequence of array-like
-         Each element of data is a sample from an underlying distribution.
+         Each element of `data` is a sample containing scalar observations from an
+         underlying distribution. Elements of `data` must be broadcastable to the
+         same shape (with the possible exception of the dimension specified by `axis`).
```

Comment:
This would be a great place to refer the reader to a note or example about using `bootstrap` with samples that contain vector observations.